### DISCUSSION TITLE
Stageplay Format

### DISCUSSION THREAD

**Discussion by calligraffiti** [eprovst/wrap#58] [Ideas]
Just curious if there are any plans to export to stageplay? I did see something about export type defaulting to screenplay, but I can't find any documentation on using --type.

**Comment by eprovst** [comment]
Hi,

Wrap can already do this :slightly_smiling_face: The type of the play is defined in the 'Title Page' section of the file, e.g.

```fountain
Title: The Night Before the Deadline
Type: Stageplay

NARRATOR
(off stage)
Our hero was a young fellow; the kind you would think still has lots of energy.
He, however, was a pro--fes--sion--al procrastinator.
```

Having to look at the documentation of 'Title Page' is indeed not self obvious. I should at some point redo the documentation, but as you might have noticed time for this project is unfortunately scarce.

The style Wrap currently uses for stageplays is not that different from the one used for screenplays, so feedback is welcome. Putting different elements on the same line (for instance, the character name and its lines), however, would at this time require quite large code changes (which currently isn't within reach for me, unfortunately), most other changes should be fairly trivial to make, though.

Thank you for your interest. Wrap actually started/escalated from some code I threw together for a school stageplay, so it's neat to see someone is interested to use it for stageplays!

**Reply by calligraffiti** [reply]
Thanks so much for your reply!

I created a quick fork, in hopes of helping you with the edit, but have no idea how to create a pull request for a wiki, so just submitted an Issue.

Hopefully it's helpful! Let me know if there's anything else you could use a hand with.

Love this tool by the way! It is hands down the best Fountain rendering tool I've come across!

Best,

Joseph
